toolchain/lower/function_context.h

```diff
@@ -126,6 +139,47 @@ class FunctionContext {
   auto FinishInit(SemIR::TypeId type_id, SemIR::InstId dest_id,
                   SemIR::InstId source_id) -> void;
 
+  auto AddCallToCurrentFingerprint(SemIR::FunctionId function_id,
```

Comment:
For this and types, had you considered adding something like a 1-byte prefix that indicates what kind of thing is being added? i.e., to force a difference even if other content matches.